ui/app/models/database/connection.js

```diff
@@ -163,9 +220,7 @@ export default Model.extend({
     // for both create and edit fields
```

Comment:
```suggestion
    // fields that are displayed for create and edit views regardless of the plugin type selected
    // does not include fields within a group as those fields are iterated over dynamically depending on what plugin the user has selected
```

Reply:
I think that might be a little confusing, because below fields are added depending on the plugin type.